assignments/number-functions/code.js

```diff
@@ -0,0 +1,31 @@
+
+const add = (n1, n2) => {
+  return n1 + n2;
+}
+const subtract = (n1, n2) => {
+  return n1 - n2;
+}
+const multiply = (n1, n2) => {
+  return n1 * n2;
+}
+const divide = (n1, n2) => {
+  return n1 / n2;
+}
+const mod = (n1, n2) => {
+  return n1 % n2;
+}
+const averageOfTwo = (n1, n2) => {
+  return (n1 + n2) / 2;
+}
+const averageOfThree = (n1, n2, n3) => {
+  return (n1 + n2 + n3) / 3;
+}
+const distance = (n1, n2) => {
+  return Math.abs(n1 - n2);
+}
+const manhattanDistance = (x1, y1, x2, y2) => {
+  return Math.abs(x1 - x2) + Math.abs(y1 - y2);
+}
+const euclideanDistance = (x1, y1, x2, y2) => {
+  return Math.sqrt((Math.abs(x1 - x2))**2 + (Math.abs(y1 - y2))**2);
```

Comment:
One mathematical note (nothing you need to change) but you don't actually need `Math.abs` here because squaring a number will always result in a positive number. You also ended up with a few more `()`s than you really need (I think you added some at some point trying to fix something and then didn't take them out when that wasn't the problem.) You can delete these:

```suggestion
  return Math.sqrt(Math.abs(x1 - x2) ** 2 + Math.abs(y1 - y2) ** 2);
```